include/openvizsla.h

```diff
@@ -25,9 +25,10 @@ struct
 #endif
 ov_packet {
 	uint8_t  magic;
-	uint16_t flags;
+	uint8_t flags;
 	uint16_t size;
-	uint32_t timestamp : 24;
+	uint16_t orig_size;
```

Comment:
I am little unsure about whole `size`/`orig_size` thing, yet I understand the purpose. The idea behind `struct ov_packet` is to have the same memory layout as in the original OV package but with easy access from C. At the same time I see that `->size` is used only once, so I don't think there is much sense to precalculate the value.

Would something like:
```C
inline uint16_t ov_packet_full_size(struct ov_packet* p) {
    return (p->flags & OV_FLAGS_HF0_TRUNC) ? OV_MAX_PACKET_SIZE : p->size;
}

//...

	    .incl_len = ov_packet_full_size(packet),
	    .orig_len = packet->size,
	    .captured = packet->data,

```
works better?

Reply:
The function should be named `ov_packet_captured_size` instead of `ov_packet_full_size`. `full` suggests the opposite of what this really is.

Reply:
Let it be `ov_packet_captured_size`

Reply:
Where should this funciton be defined at? And where should the flags be? In `openvizsla.h`?

Reply:
I think so.

`struct ov_packet` is a part of the public interface, so the flags and the function should be.